operations/mimir-mixin/alerts/alerts.libsonnet

```
@@ -557,7 +557,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
           },
           annotations: {
             message: |||
-              Ingester %(alert_instance_variable)s in %(alert_aggregation_variables)s is using too much memory.
+              Container %(alert_instance_variable)s in %(alert_aggregation_variables)s is using too much memory.
```

I would generalise it. "Instance" is also the naming we use in the doc.

```suggestion
              Ingester %(alert_instance_variable)s in %(alert_aggregation_variables)s is using too much memory.
```

---

You've accidentally put Ingester as a suggestion so I'll push a new commit.

---

Doh! Finger memory is a bad thing :(